hub/views/tools.py

```diff
@@ -0,0 +1,14 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django.views import View
+
+
+class PostcodeView(View):
+    def get(self, request: HttpRequest) -> HttpResponse:
```

Comment:
given that we're largely not using type hints elsewhere in the codebase I have a slight preference for not adding them to stray functions.